tide_site.module

```diff
@@ -588,6 +588,40 @@ function tide_site_form_alter(array &$form, FormStateInterface $form_state, $for
       }
     }
   }
+  // Add site prefix to redirect URL.
+  if ('redirect_redirect_form' === $form_id) {
+    $sites_term = 'sites';
+    // Get all primary sites ony, do not load sub-sites.
+    $term_tree = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree(
```

Comment:
I think if we're breaking this over multiple lines, it aids readability if you put line breaks between the chained method calls, rather than in the params. I found this a bit hard to read and initially confusing. 

I would suggest this is easier to read and follow for me:

```php
    // Get the primary Sites only. Do not load child Sites sections.
    $vocabulary = 'sites';
    $term_tree = \Drupal::entityTypeManager()
      ->getStorage('taxonomy_term')
      ->loadTree($vocabulary, 0, 1, TRUE);

```